scripts/workers/rl_submission_worker.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import os
+import signal
+import time
+
+from .worker_util import (
+    EvalAI_Interface
+)
+
+from kubernetes import client, config
+
+
+class GracefulKiller:
+    kill_now = False
+
+    def __init__(self):
+        signal.signal(signal.SIGINT, self.exit_gracefully)
+        signal.signal(signal.SIGTERM, self.exit_gracefully)
+
+    def exit_gracefully(self, signum, frame):
+        self.kill_now = True
+
+
+logger = logging.getLogger(__name__)
+
+AUTH_TOKEN = os.environ.get("AUTH_TOKEN", "x")
```

Comment:
@fristonio @vkartik97 is there a specific reason to have these as envs. Can we instead have these as arguments to the main? These can be passed while starting the container. Let me know your thoughts.